data/autoyast_sle15/create_hdd/create_hdd_ha_sles_12.xml.ep

```diff
@@ -0,0 +1,400 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
the name of the file should be `sles12sp4` instead of `sles_12` we don't know if it will work for sles12sp5.

Reply:
Updated the name to sles_12sp4. For later PR we can verify this and if works for sles12sp5 then we can update the name then.